nservicebus/upgrades/5to6.md

```diff
@@ -105,6 +105,13 @@ NServiceBus will automatically make correlated saga properties unique without th
 Saga persisters (`ISagaPersister`) and finders (`IFindSagas`) introduce a new parameter `SagaPersistenceOptions`. This parameter give access to the saga metadata and pipeline context. This enables  persisters and finders to manipulate everything that exists in the context during message pipeline execution. For more information see [Sagas](/nservicebus/sagas/) and [Complex saga finding logic](/nservicebus/sagas/saga-finding.md).
 
 
+## Subscriptions
+
+### Subscription storage
+`ISubscriptionStorage` has been split into three interfaces, `ISubscriptionStorage`, `IInitializableSubscriptionStorage` and `IQuerySubscriptions`, to properly separate those storage concerns. `ISubscriptionStorage` and `IQuerySubscriptions` must be implemented to have a fully functional subscription infrastructure. `IInitializableSubscriptionStorage` is only necessary when the subscription storage needs to be initialized.
```

Comment:
So while this is good from the upgrade perspective where is the full doco for someone who starts on version 6. Where do they read about what these interfaces are for?